docs/what-is-bitcoin.md

```diff
@@ -0,0 +1,71 @@
+## Overview
+
+Bitcoin is a peer-to-peer digital currency and network. Bitcoins can be sent and received through
+the internet, and aren't centrally controlled by anyone. Unlike using a bank, this means nobody
+can seize, block or take control of your money. Instead, Bitcoin is run and verified collectively by
+its users, as a peer-to-peer network. Let's get started and explain the history behind Bitcoin
+and how it works.
+
+## Background and transactions
+
+Bitcoin was first described in the Bitcoin whitepaper, by Satoshi Nakamoto, its first developer.
+Before Bitcoin, digital currencies and payment systems all relied on trust, by either trusting a
+company, or an individual. This made it easy for these trusted persons to block people's money,
+or succumb to Government intervention. Satoshi figured out a way to overcome that, by inventing
+a data structure called the blockchain, which is distributed between all users in a peer-to-peer network.
+
+Users of the Bitcoin network create transactions, and then digitally sign them with cryptographic
+signatures. A transaction contains all the information necessary to send the bitcoins to its recipient.
+Once the transaction is ready, it gets broadcasted to the Bitcoin peer-to-peer network so that it can
+be included into the blockchain by the Bitcoin miners, a group of users and businesses who use special
+hardware to solve really complex puzzles to secure transactions and extend the blockchain. But what
+really is the blockchain?
+
+## Blockchain
+
+The blockchain is a chain of blocks, created by the Bitcoin miners by solving computationally
+difficult puzzles using their specialised hardware. Each block is generated approximately every
```

Comment:
The first sentence here repeats some of the same information as the penultimate sentence of the previous paragraph and so one of those sentences can probably have some words cut losslessly.